powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/CredentialCounterService.java

```diff
@@ -95,7 +95,6 @@ public UpdateCounterResponse updateCredentialCounter(UpdateCounterRequest reques
      * @throws InvalidRequestException Thrown when request is invalid.
      */
     public void updateCredentialCounter(CredentialEntity credential, AuthenticationResult authenticationResult) throws InvalidRequestException {
-        credential.setAttemptCounter(credential.getAttemptCounter() + 1);
```

Comment:
where did this goes?

Reply:
This call was duplicate, check other usages of `credential.setAttemptCounter()`. The attempt counter is increased earlier during authentication right after request validation.